Packages/ClientRuntime/Sources/Waiters/Acceptor.swift

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+extension WaiterConfig {
+    /// `Acceptor` is a Swift-native equivalent of Smithy acceptors:
+    /// https://awslabs.github.io/smithy/2.0/additional-specs/waiters.html#acceptor-structure
+    public struct Acceptor {
+
+        public typealias Matcher = (Input, Result<Output, Error>) -> Bool
+
```

Comment:
lol this is a dumb linter rule.

Speaking of dumb linter rules, we don't seem to have any `.swiftlint` file in this project.  Wonder what rules we are using?

Reply:
https://github.com/awslabs/smithy-swift/blob/main/Packages/.swiftlint.yml

either update the rule or fix the issue.

Reply:
Rule removed.  This is better left to the dev case-by-case IMO